docs/upload.rst

```diff
@@ -73,17 +74,6 @@ For example, here's the contents of a symbols ZIP file from a Firefox build::
     xpcshell/B95F899A59E247055C0ED7883D0235A40/xpcshell.sym
 
 
-Permissions and auth token
---------------------------
-
-Uploading requires two things:
-
-1. an account with upload permissions
-2. an auth token for that account with upload permissions
-
-The auth token is used in the HTTP POST to authenticate the upload.
```

Comment:
This part was in this document twice, so I nixed it from here figuring the other section is more fleshed out.